### PR TITLE
Updated Docs

### LINKED ISSUE #147: Add link from website/docs back to github

### PR DESCRIPTION
**Pull request by SebastiaanNijland**
Fixes #147 

- Added link to github (fixed to sidebar bottom)
- Added meta title

### CODE REVIEW

**Comment by joepuzzo** [comment]
Make sure you exploratory test this by running `npm start` locally. The new anchor tag you added is stretching across the whole page. I would instead just add the link back to git at the bottom of the existing nav as another nav item and get rid of that fixed class.

File: dev/SideNav/SideNav.scss [hunk 1]
```diff
@@ -7,6 +7,9 @@
 }
 
 .sticky-sidebar {
-  position:fixed;
+  position: fixed;
   max-width: 20%;
+  .fixed-bottom {
```
Comment (joepuzzo) [inline dev/SideNav/SideNav.scss]: I would do away with this and just add the link to git as another nav item.

**Comment by SebastiaanNijland** [comment]
Will do, thanks :)

**Comment by joepuzzo** [comment]
Looks good!

**Pull request merged**